### ISSUE TITLE
Wrong joint printing

### ISSUE THREAD

**Issue by gabrielebndn** [stack-of-tasks/pinocchio#803]
The following code:
```
  Model model;
  buildModels::humanoid(model);
  Data data(model);
  std::cout << model.joints[1] << std::endl;
```

prints
```
JointModelFreeFlyer
  index: 18446744073709551615
  index q: -1
  index v: -1
  nq: 7
  nv: 6
```

which is wrong since it should be
```
JointModelFreeFlyer
  index: 1
  index q: 0
  index v: 0
  nq: 7
  nv: 6
```

This is weird because
```
std::cout << model.joints[1].id() << std::endl;
std::cout << model.joints[1].idx_q() << std::endl;
std::cout << model.joints[1].idx_v() << std::endl;
```
prints  correct values
```
1
0
0
```

The same happens in Python.
Although I think this problem may be fairly easy to solve, I do not understand why it happens.

**Comment by jcarpent** [comment]
I know the origin of this issue. I will solve it asap.
@gabrielebndn Thanks for raising it.

**Comment by jcarpent** [comment]
Fixed by #804.
